tcptracer-bpf.h

```diff
@@ -79,16 +79,23 @@ struct tracked_socket {
 
 #define EVENT_HTTP_RESPONSE 1
 #define EVENT_MYSQL_GREETING 2
+#define EVENT_HTTP_RESPONSE_V6 3
+#define EVENT_MYSQL_GREETING_V6 4
+
+struct connections {
```

Comment:
shouldnt it be a union?

Reply:
It was, but that complicated the golang implementation a bit.  I can take another stab at it, I think I understand it better now. Should I @Q69K ?

Reply:
Did it anyway.